File: java/events/Ready.java
package events;

import javax.annotation.Nonnull;
import net.dv8tion.jda.api.hooks.ListenerAdapter;
import net.dv8tion.jda.api.events.session.ReadyEvent;
import structures.Discord;

public class Ready extends ListenerAdapter {
    protected Discord discord;

    public Ready(Discord discord) {
        this.discord = discord;
    }

    @Override
    public void onReady(@Nonnull ReadyEvent event) {
        var discord = event.getJDA().getSelfUser();
        System.out.printf("Logged in as %s!\n", discord.getName());
    }   
}

File: kotlin/bin/main/events/Ready.kt
package events

import net.dv8tion.jda.api.hooks.ListenerAdapter
import net.dv8tion.jda.api.events.session.ReadyEvent
import structures.Discord

class Ready(private val discord: Discord) : ListenerAdapter() {
    override fun onReady(event: ReadyEvent) {
        val selfUser = event.jda.selfUser
        println("Logged in as ${selfUser.name}!")
    }
}